web-client/src/views/TrialSessions/EditTrialSession.tsx

```diff
@@ -88,7 +98,7 @@ export const EditTrialSession = connect(
               data-testid="submit-edit-trial-session"
               type="submit"
               onClick={() => {
-                updateTrialSessionSequence();
+                displayTrialSessionLocationChangeModalSequence();
```

Comment:
Can we rename this since the user isn't necessarily editing a trial session location? (Maybe something like `handleEditedTrialSessionSequence`?)